oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/web/client/ClientRegistrationIdProcessorTests.java

```diff
@@ -39,6 +39,8 @@
  */
 class ClientRegistrationIdProcessorTests {
```

Comment:
Add tests that verify failure in the event that there are duplicate annotations. This should be handled by the `SecurityAnnotationScanners.requireUnique`. An example would be `AggregateService.toTest()` should fail due to duplicate annotations at the same level.

```
@ClientRegistrationId("a")
interface AService {}

@ClientRegistrationId("b")
interface BService {}

interface AggregateService extends AService, BService {
	void toTest();
}
```